libevm/EVMC.cpp

```diff
@@ -10,15 +10,33 @@ namespace dev
 {
 namespace eth
 {
-EVM::EVM(evmc_instance* _instance) noexcept : m_instance(_instance)
+namespace
 {
-    assert(m_instance != nullptr);
-    assert(evmc_is_abi_compatible(m_instance));
+evmc_revision toRevision(EVMSchedule const& _schedule) noexcept
+{
+    if (_schedule.haveCreate2)
+        return EVMC_CONSTANTINOPLE;
```

Comment:
Is there a "Constantinople2" hardfork in aleth yet?

Reply:
Not yet.